src/query-hoc.tsx

```diff
@@ -58,6 +58,7 @@ export function query<
 >(
   document: DocumentNode,
   operationOptions: OperationOption<TProps, TData, TGraphQLVariables, TChildProps> = {},
+  logUnhandled: boolean = false,
```

Comment:
should this be like `processingOptions.logUnhandled`?

Reply:
Just suggesting to use named parameters in a new options key instead of positional.

Reply:
@timbotnik ah! Well the plan is for this to be removed in the very near term with a 3.0 of react-apollo (no API change, just removal of deprecated options). So I think having it an internal positional argument is good because it prohibits doing more things in this position